### PR TITLE
removed unregistration of autoloaders other than composer

### PR DESCRIPTION
**Pull request by f3nrir92**
There was a problem working with Sympfony, when used Swift mailer: Swift's autoloader does not calls due to unload.

### CODE REVIEW

**Comment by vladimmi** [comment]
Hello and thanks for your interest to this small tool. Cant understand one thing. Swift Mailer docs says:

> If you are using Composer, Swift Mailer will be automatically autoloaded.

As this loader depends on Composer and wraps it, I believe no other autoloader calls should be needed. What is the autoloader you want to use?

**Comment by f3nrir92** [comment]
Yes, it is. But Swift also register Swift::autoload for loading it's classes.

**Comment by vladimmi** [comment]
OK, just a moment to test it (maybe I really should add tests for such things...)

File: src/Loader.php [hunk 1]
```diff
@@ -72,7 +72,10 @@ public function __construct(ClassLoader $loader, $params = [])
         //unregister composer
         $loaders = spl_autoload_functions();
         foreach ($loaders as $loader) {
-            spl_autoload_unregister($loader);
+	    // we need to replace only composer
```
Comment (vladimmi) [inline src/Loader.php]: indent should be spaces, one tab makes comment aligned different to code around

File: src/Loader.php [hunk 2]
```diff
@@ -72,7 +72,10 @@ public function __construct(ClassLoader $loader, $params = [])
...
+            if(is_array($loader) && $loader[0] instanceof ClassLoader) {
```
Comment (vladimmi) [inline src/Loader.php]: there should be space between "if" and "("

File: src/Loader.php [hunk 3]
```diff
@@ -72,7 +72,10 @@ public function __construct(ClassLoader $loader, $params = [])
         //unregister composer
         $loaders = spl_autoload_functions();
         foreach ($loaders as $loader) {
```
Comment (vladimmi) [inline src/Loader.php]: maybe it's better to name it not "$loader" to not mask function parameter with same name. My fail - can you rename it to something like "$l"?

**Comment by vladimmi** [comment]
Can you make some corrections, please? Formatting mostly, added line comments to your commit.

**Comment by f3nrir92** [comment]
Done. 

P.S. Will never write code outside of IDE =)

**Comment by vladimmi** [comment]
Thanks for your efforts :)

**Comment by f3nrir92** [comment]
You're welcome ))

**Pull request merged**